File: API/python/examplebot.py
#!/usr/bin/env python

import sys, os, random, time
from GoBot import GoBot

class ExampleBot(GoBot):
	def onMyTurn(self,player,boardState):
		time.sleep(1)
	
		x = random.randint(0,8)
		y = random.randint(0,8)
		
		while boardState[x][y] != 0:
			x = random.randint(0,8)
			y = random.randint(0,8)
			
		p = random.randint(0,94);
			
		if p != 3:
			p = 0
		else:
			p = 1
	
		return x, y, p # here is my move
	
gobot = ExampleBot()
gobot.start()

File: API/java/ExampleBot.java
import java.util.Random;

public class ExampleBot extends GoBot {
	public ExampleBot() {}
	
	@Override
	public Move onMyTurn(int pnum, int[][] boardState) {
		Random random = new Random();
		
		int x, y;
		do {
			x = random.nextInt(9);
			y = random.nextInt(9);
		} while(boardState[x][y] != 0);
		
		return new Move(x,y,0);
	}
	
	public static void main(String[] args) {
		ExampleBot bot = new ExampleBot();
		bot.start();
	}
}
